src/phoenix/trace/fixtures.py

```diff
@@ -146,6 +214,50 @@ def load_example_traces(use_case: str) -> TraceDataset:
     return TraceDataset(json_lines_to_df(download_traces_fixture(fixture)))
 
 
+def get_dataset_fixtures(
+    use_case: str,
+) -> Iterable[DatasetFixture]:
+    return (fixture.load() for fixture in get_trace_fixture_by_name(use_case).dataset_fixtures)
+
+
+def send_dataset_fixtures(
+    endpoint: str,
+    fixtures: Iterable[DatasetFixture],
+) -> None:
+    sleep(5)  # wait for server to start
```

Comment:
Suggestion: You can do a while loop pinging `/healthz` every second.

Reply:
oh cool. thx